pkg/driver/controller.go

```diff
@@ -205,6 +218,18 @@ func (cs *controller) CreateVolume(
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	_, state, err = zfs.GetZFSVolumeStatus(req.Name)
+
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "createvolume: failed to fetch the volume %v", err.Error())
+	}
+
+	if state == zfs.ZFSStatusPending {
+		return nil, status.Errorf(codes.Internal, "volume %s is being created", volName)
```

Comment:
Can this be warning.

Reply:
this is not the log. The is the error returned to the CSI provisioner so that it can retry the volume provisioning.